[PATCH] don't append _d to library name when Qt is built with debug support

qtuio appends _d to the library name when Qt was built with debug support,
breaking linking step for examples and staging/target install.

There's no real advantage to the _d suffix, so simply fix it by removing
the logic appending _d.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 src/qTUIO.pro |    5 -----
 1 file changed, 5 deletions(-)

Index: qtuio-abe4973ff6/src/qTUIO.pro
===================================================================
--- qtuio-abe4973ff6.orig/src/qTUIO.pro
+++ qtuio-abe4973ff6/src/qTUIO.pro
@@ -16,11 +16,6 @@
 win32:LIBS += ws2_32.lib \
     winmm.lib
 
-# Changes the name of the target, when is debug mode
-CONFIG( debug, debug|release ) { 
-    TARGET = $${TARGET}_d
-    BUILD_NAME = debug
-}
 CONFIG( release, debug|release ):BUILD_NAME = release
 
 # Temporary folders for the auxiliar files
